build_tools/third_party/s3_management/manage.py

```diff
@@ -37,10 +37,15 @@
 ACCEPTED_FILE_EXTENSIONS = ("whl", "zip", "tar.gz")
 PREFIXES = [
     "v2/gfx110X-dgpu",
+    "v2-staging/gfx110X-dgpu",
     "v2/gfx1151",
+    "v2-staging/gfx1151",
     "v2/gfx120X-all",
+    "v2-staging/gfx120X-all",
     "v2/gfx94X-dcgpu",
+    "v2-staging/gfx94X-dcgpu",
     "v2/gfx950-dcgpu",
+    "v2-staging/gfx950-dcgpu"
```

Comment:
```suggestion
    "v2/gfx110X-dgpu",
    "v2/gfx1151",
    "v2/gfx120X-all",
    "v2/gfx94X-dcgpu",
    "v2/gfx950-dcgpu",
    "v2-staging/gfx110X-dgpu",
    "v2-staging/gfx1151",
    "v2-staging/gfx120X-all",
    "v2-staging/gfx94X-dcgpu",
    "v2-staging/gfx950-dcgpu"
```

Maybe reorder to have all v2 and v2-staging grouped?